tests/test_CapData.py

```diff
@@ -1601,9 +1613,9 @@ def test_specify_ghi_col(self, nrel_clear_sky):
         for i, col in enumerate(nrel_clear_sky.data_filtered.columns):
             assert col == nrel_clear_sky.data.columns[i]
 
-    def test_no_clear_sky(self, nrel_clear_sky):
-        with pytest.warns(UserWarning):
-            nrel_clear_sky.filter_clearsky(window_length=2)
+    #def test_no_clear_sky(self, nrel_clear_sky):
+    #    with pytest.warns(UserWarning):
+    #        nrel_clear_sky.filter_clearsky(window_length=2)
```

Comment:
This test is failing, I think due to some change in scipy as well. I've commented it out for now, but TBD what to do with it in the long run.  If we consider it a bug to be fixed in pvlib then maybe OK to not make any changes here in pvcaptest.

Reply:
@kandersolar, thank you for the pull request! I'm away from my computer this weekend, but I'll try to review and merge soon.

Reply:
Ok, a bit of background: `pvlib.clearsky.detect_clearsky` uses `scipy` for an optimization.  That optimization can fail under certain conditions like the ones constructed by this test, but scipy until recently returned plausible but garbage answers in these failures.  Now it returns NaN, which causes a subsequent line in pvlib to fail.  This is fixed as part of https://github.com/pvlib/pvlib-python/pull/1784, where the "fix" is to raise an error if the optimization fails.

I'm not sure what is best to do here in `pvcaptest`.  I don't immediately see an easy way to bridge the behavior gaps between the relevant versions of scipy and pvlib.  An easy option is to wait for the next pvlib release, which should include that fix, and increase pvcaptest's minimum allowed pvlib to that version, but I'm not sure requiring the absolute latest pvlib is palatable.